Formula/arangodb.rb

```diff
@@ -33,6 +59,7 @@ def install
         -DCMAKE_INSTALL_DATAROOTDIR=#{share}
         -DCMAKE_INSTALL_SYSCONFDIR=#{etc}
         -DCMAKE_INSTALL_LOCALSTATEDIR=#{var}
+        -DCMAKE_OSX_DEPLOYMENT_TARGET=#{MacOS.version}
```

Comment:
Is this required? Cause this won't work on linux

Reply:
Otherwise the link step will produce a lot of warnings. It will build without this option but it looks a bit scary in the log file.

Reply:
@SMillerDev I have removed the unnecessary require and used the git commit.

I have not found an example on how to express that it fails with GCC 7.3.0 & 8.2.0, but works with 7.40. Shall I just ignore 7.4.0?

I have also not found an example on how to deal with the MacOS.version. The other formulas I looked at, use the same string.

Please advise.

Thanks a lot Frank